pexconn.go

```diff
@@ -100,8 +109,10 @@ func (s *pexConnState) Recv(payload []byte) error {
 	peers.AppendFromPex(rx.Added6, rx.Added6Flags)
 	peers.AppendFromPex(rx.Added, rx.AddedFlags)
 	s.dbg.Printf("adding %d peers from PEX", len(peers))
-	s.torrent.addPeers(peers)
-	// s.dbg.Print("known swarm now:", s.torrent.KnownSwarm())
+	if len(peers) > 0 {
+		s.torrent.pex.rest = time.Now().Add(pexInterval)
+		s.torrent.addPeers(peers)
```

Comment:
Would prefer not to guard this call, unless there's some special treatment of 0 peers in addPeers.

Reply:
This guard is mainly to avoid rewinding the cool down timer upon receipt of the PEX messages which carry drops only.